### DISCUSSION TITLE
userid만으로 강퇴 하려면

### DISCUSSION THREAD

**Discussion by yje0404N** [storycraft/node-kakao#797] [Q&A]
if (data.text=="!test"){
      channel.kickUser({ "userId": new Long("1234567898") });
    }

유저정보를 따로가져오지않고 userid로만 따로 써서 강퇴하려면 
어떻게 써줘야 하나요??

위 코드대로라면 에러코드가 떠서..
Client error!! err: ReferenceError: Long is not defined

**Comment by benesiaxd** [comment]
if (data.text=="!test"){
channel.kickUser({ "userId": 1234567898 });
}
